tests/python/gaia-ui-tests/gaiatest/apps/search/app.py

```diff
@@ -33,15 +40,9 @@ def go_to_url(self, url):
 
     @property
     def history_items_count(self):
+        Wait(self.marionette).until(expected.element_present(*self._history_locator))
```

Comment:
Oh wait, I found out why. You might want to make sure the history is loaded an there's no item in the history. Maybe that's worth to be mentioned in a comment